admin/src/services/presetWidget.js

```diff
@@ -1,39 +1,45 @@
 import { createApiGet, createApiPost } from "../utils";
-import { BASE_URL_BACKEND } from "../config";
+import { ADMIN_ACTIONS_WS_URL, BASE_URL_BACKEND } from "../config";
 
 export class PresetWidgetService {
     constructor(apiPath, errorHandler) {
         this.apiUrl = BASE_URL_BACKEND + apiPath;
```

Comment:
Probalby shouldn't just append strings when constructing a path
Or only do this in one place, not in all Services
Or just replace `apiPath` with widgetType or some other string